app/Http/Requests/Api/StorePaletteRequest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Http\Requests\Api;
+
+use App\Http\Requests\ApiBaseRequest;
+use Illuminate\Contracts\Validation\ValidationRule;
+use Illuminate\Support\Facades\Auth;
+
+class StorePaletteRequest extends ApiBaseRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     */
+    public function authorize(): bool
+    {
+        return Auth::guard('sanctum')->check();
+    }
```

Comment:
This request passes `auth:sanctum` middleware so this statement always return `true`